test/jdk/java/net/httpclient/PlainProxyConnectionTest.java

```diff
@@ -51,18 +53,32 @@
 
 /**
  * @test
- * @bug 8230526
+ * @bug 8230526 8342811
  * @summary Verifies that PlainProxyConnections are cached and reused properly. We do this by
  *          verifying that the remote address of the HTTP exchange (on the fake proxy server)
- *          is always the same InetSocketAddress.
- * @modules jdk.httpserver
- * @run main/othervm -Djdk.tracePinnedThreads=full PlainProxyConnectionTest
+ *          is always the same InetSocketAddress. Logging verbosity is increased to aid in
+ *          diagnosis of intermittent failures reported in 8342811.
```

Comment:
Not a big deal but there's no need to add that to the test summary.

Reply:
Removed in 8cec4a7769f71dc0ca44e8c9e0bbfadcb0531927. (I had kept it around so that if the issue pops up again and we figure out how to fix it, we will have a handle to go to the ticket and resolve it. But there are other, maybe better, ways to do this, I guess.)

Reply:
What I meant is that there was no need to modify the test summary to explain that logging has been enabled. I mean the test summary is there to explain what the test is trying to verify. Its not a changelog ;-)